services/cloudformation/src/it/java/software/amazon/awssdk/services/cloudformation/SendEmptyListIntegrationTest.java

```diff
@@ -83,9 +83,6 @@ public void setup() {
     @After
     public void tearDown() {
         cf.deleteStack(DeleteStackRequest.builder().stackName(stackName).build());
-        cf.waiters()
```

Comment:
This is on the teardown method so don't think that we really need to wait for it to get deleted as long as the original request is received.